fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/remote/MessageSender.java

```diff
@@ -60,4 +60,6 @@ public interface MessageSender extends Disposable {
    * @return a {@link Completable} the will complete once the unsubscribe is finished
    */
   Completable unsubscribe(Channel channel);
+
+  Connection getConnection();
```

Comment:
Why do you need this ? We decided to abstract the connection away when implementing this part of the app. Is this really necessary ?

Reply:
I add a Listener to avoid for the subscribe and catchup process to begin before the change of connection is finished